analytics_fetcher/support/utils.py

```diff
@@ -4,7 +4,8 @@
 
 def base64_encode(bs):
     """Convert bytes to base64"""
-    return b64encode(bs).replace("\n", "")
+    return b64encode(bs).replace(b"\n", b"")
+
 
 def base64_decode(s):
```

Comment:
Could we rename this param please `string` or at least `str` ?

Reply:
I've kept it consistent with the above and kept it as 'byte_string', all done ✅